src/pages/embed.vue

```diff
@@ -190,6 +193,15 @@ const route = useRoute()
 const timePeriod = ref(simulationArea.timePeriod)
 const clockEnabled = ref(simulationArea.clockEnabled)
 
+// Embed user preferences
+const theme = ref(route.query.theme)
+const hasDisplayTitle = ref(route.query.display_title ? route.query.display_title === 'true' : false);
+const hasClockTime = ref(route.query.clock_time ? route.query.clock_time === 'true' : true);
+const hasFullscreen = ref(route.query.fullscreen ? route.query.fullscreen === 'true' : true);
+const hasZoomInOut = ref(route.query.zoom_in_out ? route.query.zoom_in_out === 'true' : true);
```

Comment:
```suggestion
const theme = ref(route.query.theme)
const hasDisplayTitle = ref(route.query.display_title ? route.query.display_title === 'true' : false);
const hasClockTime = ref(route.query.clock_time ? route.query.clock_time === 'true' : true);
const hasFullscreen = ref(route.query.fullscreen ? route.query.fullscreen === 'true' : true);
const hasZoomInOut = ref(route.query.zoom_in_out ? route.query.zoom_in_out === 'true' : true);
```

- Try to make computed, ref does not make sense
- make is type
- try to make it boolean by casing